.github/workflows/licensecheck.yml

```diff
@@ -1,6 +1,12 @@
-name: Request license-review
+# This workflow will check for Maven projects if the licenses of all (transitive) dependencies are vetted.
+
+name: License vetting status check
```

Comment:
This is now misleading as it also handles the previous `Request license-review`

Reply:
Then lets just name it `License status`.

Reply:
This is still wrong, requesting a review is not a status, also I suspect that now the check will maybe become green if requesting succeeds... I still see no benefit in mixing these two concerns into one workflow run that makes it more complex to manage and understand and test.

e.g. previously if the check failed one could rerun it, what will happen now is unclear, will it repeat the check of licenses or will it repeat the request for license and all that kind of uncertainties.